client/landing/stepper/declarative-flow/flows/ai-site-builder/ai-site-builder.ts

```diff
@@ -141,8 +147,12 @@ const aiSiteBuilder: Flow = {
 
 						if ( prompt ) {
 							promptParam = `&prompt=${ encodeURIComponent( prompt ) }`;
+						} else if ( window.sessionStorage.getItem( 'stored_ai_prompt' ) ) {
```

Comment:
This brute forcing of carrying the prompt in session storage is needed because Github login redirect flow is loosing all query string here:

https://github.com/Automattic/wp-calypso/blob/ccb32fe85c15ad18d6b501cf6dcb9427f7e1406e/client/components/social-buttons/github.tsx#L159